### ISSUE TITLE
Colorpicker: ARIA attribute must conform to valid value

### ISSUE THREAD

**Issue by jcjuett** [salesforce/design-system-react#2014]
https://design-system-react-co-pr-2004.herokuapp.com/?selectedKind=SLDSColorPicker&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Issue: Automated aXe test fails ARIA attribute. 
expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('#color-picker-summary-input-default-color-picker') to have no violations:

   ` <input aria-describedby="color-picker-summary-error-default-color-picker" class="slds-input" id="color-picker-summary-input-default-color-picker" type="text" value="">`

    Received:

    "ARIA attributes must conform to valid values (aria-valid-attr-value)"

This is most likely due to the ID that is being pointed to not being on the page when the markup audit occurs. The solution is to only add the ID when the menu/popover is on the page. This typically done through a conditional called isOpen.

Solution: See https://github.com/salesforce/design-system-react/pull/2004 as possible solution.

**Comment by interactivellama** [comment]
@jcjuett This was merged in today and should be available for auditing at http://design-system-react-components.herokuapp.com/?selectedKind=SLDSColorPicker&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel shortly.
